src/components/SchemaRow.tsx

```diff
@@ -55,7 +55,7 @@ export const SchemaPropertyRow: typeof SchemaRow = ({ node, onGoToRef, rowOption
           style={{
             width: ICON_DIMENSION,
             height: ICON_DIMENSION,
-            ...(isRefNode(schemaNode)
+            ...(isRefNode(schemaNode) && Tree.getLevel(node) === 0
```

Comment:
@P0lip this refers to last fix for ref nodes caret. I suggest we do that only for level 0 nodes otherwise they may bey shifted

![Screenshot from 2020-03-31 16-19-40](https://user-images.githubusercontent.com/14196079/78037466-e3dfa080-736b-11ea-93e1-bb26b4bdbeda.png)

Reply:
Ah yes, agreed.